### ISSUE TITLE
scream_arch.cpp does not report the correct data

### ISSUE THREAD

**Issue by ambrad** [E3SM-Project/scream#487]
scream_arch.cpp does not report the correct data. The packsize line was commented out, and the SCREAM_FPE line reports "off" when it is actually "on". I believe both of these are associated with a change to using an EKAT config file rather than the scream one.

Reproduce by setting up a packsize=1 debug build and then looking at config_string output when running a test.

**Comment by bartgol** [comment]
There are two issues here, the pack size and the FPE one.

I originally removed the pack size output cause I was planning to remove any pre-defined pack size from ekat (leaving it generic in terms of pack size), but had to keep pack size info in it, in order to build some pack utilities (like `smallize`), and in order to get `LinInterp` to build (since it does require the POSSIBLY_NO_PACK_SIZE macro). I am still in favor of removing hard-coded Pack sizes in Ekat, so that it can become generic. The concept of a "small" pack, for instance, should not belong to Ekat, but to Scream, or, even better, to each individual parametrization (which may have different needs).

Regarding FPEs, the same thing happened: I removed hard coded FPE setting from Ekat, and left it in scream. So I guess for this one, the correct behavior should be for scream to print scream-specific settings right after the call to `initialize_scream_session` (which is a function in Ekat now). Perhaps it's about time we remove scream from Ekat.

**tl;dr:** I think the behavior of scream_arch.cpp is the right one (some settings do not belong in ekat). However, if we don't inject our settings in ekat, I have no idea how to print scream settings from Ekat initialization (inside, e.g., `ekat_catch_main.cpp`). I need to give it some thought and come up with a solution that a) prints all our settings to screen, and b) does not require Ekat to be aware of downstream app setting at config time.

**Comment by bartgol** [comment]
This has been fixed at some point (with one of the ekat updates I think).
